### PR TITLE
chore(combobox, date-picker, input-date-picker, input): fix misc typos in stories, component props

### PR DESCRIPTION
**Pull request by caripizza**
**Related Issue:** n/a

## Summary
Fixes a few typos I noticed while perusing our Storybook:
* removes extra quotation mark and references battery3 icon in combobox "Single" story
* changes date-picker and input-date-picker's default locale prop to use "en", updates stories to use locale and corrects range "end" dates
* fixes theme prop typo in calcite-input readme

### CODE REVIEW

**Review by driskull — APPROVED** [review]
👍 nice

File: src/components/calcite-combobox/calcite-combobox.stories.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const Single = (): string => html`
       <calcite-combobox-item icon="article" value="article" text-label="Article"></calcite-combobox-item>
       <calcite-combobox-item icon="attachment" value="attachment" text-label="Attachment"></calcite-combobox-item>
       <calcite-combobox-item icon="banana" value="banana" text-label="Banana"></calcite-combobox-item>
-      <calcite-combobox-item icon="battery" value="-battery" text-label="Batterycharging"></calcite-combobox-item>
+      <calcite-combobox-item icon="battery3" value="battery" text-label="Batterycharging"></calcite-combobox-item>
```
Comment (eriklharper) [inline src/components/calcite-combobox/calcite-combobox.stories.ts:66]: I thought that `-` looked fishy there.  Thanks for whipping this up!

**Review by eriklharper — APPROVED** [review]

**Comment by paulcpederson** [comment]
Niiiiiiiiiice. Thanks @caripizza  I had been meaning to look at why the combobox story was being weird

**Review by paulcpederson — APPROVED** [review]

**Comment by julio8a** [comment]
Awesome! can we merge this one?

**Pull request merged**